web/css/style.css

```diff
@@ -203,86 +403,192 @@ li .tagline {
 }
 
 .fork-and-edit {
-  background: url("../img/github.gif") no-repeat scroll 0 0 transparent;
-  font-style: italic;
-  height: 40px;
-  margin: 40px auto 0;
-  max-width: 580px;
-  padding-top: 5px;
-  padding-left: 40px;
+    background: url("../img/github.gif") no-repeat scroll 0 0 transparent;
+    font-style: italic;
+    height: 40px;
+    margin: 40px auto 0;
+    max-width: 580px;
+    padding-top: 5px;
+    padding-left: 40px;
 }
 
 table {
-  width: 100%;
+    width: 100%;
 }
 
 tr {
-  border-bottom: 1px solid #999;
+    border-bottom: 1px solid #999;
 }
 
 tr:last-child {
-  border-bottom: none;
+    border-bottom: none;
 }
 
 th, td {
-  text-align: left;
-  padding: 0.5em;
+    text-align: left;
+    padding: 0.5em;
```

Comment:
Why are you changing the indentation used in the CSS file ?

Reply:
As explained below: reason why + revert done.